services_app/src/main/res/layout/conflict_resolver_field_list.xml

```diff
@@ -18,36 +19,33 @@
         android:layout_height="wrap_content"
         android:orientation="vertical" >
 
-        <androidx.appcompat.widget.AppCompatButton
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/take_server"
             android:text="@string/conflict_take_server_updates"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="24dip"
-            android:layout_weight="50"
+            android:layout_marginBottom="24dp"
```

Comment:
Any reasons for adding a layout_marginBottom and setting it to 24dp?

Reply:
The only reason is to use dp instead of dip. It makes it consistent across the files, as in other files dp has been used.

Reply:
Great